docs/guide/slots.md

```diff
@@ -39,7 +43,7 @@ To render a `renders_many` slot, iterate over the name of the slot:
 ```erb
 <%# index.html.erb %>
 <%= render BlogComponent.new do |c| %>
-  <% c.header(classes: '') do %>
```

Comment:
At this point in the docs, component slots haven't been introduced, so passing these arguments doesn't do anything.